README.md

```diff
@@ -1,25 +1,73 @@
 # Canonical Kubernetes Snap
 [![End to End Tests](https://github.com/canonical/k8s-snap/actions/workflows/e2e.yaml/badge.svg)](https://github.com/canonical/k8s-snap/actions/workflows/e2e.yaml)
 
-Canonical Kubernetes is an opinionated Kubernetes delivered by snaps. The focus is on:
-- simplified almost zero operations and
-- enhanced security posture on any infrastructure
+**Canonical Kubernetes** is the fastest, easiest way to deploy a fully-conformant Kubernetes cluster. Harnessing pure upstream Kubernetes, this distribution adds the missing pieces (e.g. ingress, dns, networking) for a zero-ops experience. For single-node clusters it can be deployed with two commands. Add new nodes with just two more. 
 
+Get started in just two commands:
```

Comment:
This reads a bit weird.
It either say:
```
Get started with:
sudo snap install k8s --classic
And bring up the cluster:
sudo k8s bootstrap
```
or

Get started in just two commands:
```
sudo snap install k8s --classic
sudo k8s bootstrap
```

Reply:
I like the second suggestion better.